File: Levels/LevelItems/Level.cs
using Mario.Levels.LevelItems;
using System.Collections.Generic;

namespace Mario.Levels.Level
{
    public class Level
    {
        public string pathToSpriteJson { get; set; }
        public string level { get; set; }
        public int width { get; set; }
        public int height { get; set; }
        public int timeLimit { get; set; }
        public string song { get; set; }
        public LevelHero hero { get; set; }
        public List<LevelEnemy> enemies { get; set; }
        public List<LevelItem> items { get; set; }
        public List<LevelItemSection> itemSections { get; set; }
        public List<LevelBlockSection> blockSections { get; set; }
        public List<LevelBlock> blocks { get; set; }
        public List<LevelPipe> pipes { get; set; }

    }
}

File: Mario-Level-Creator/src/models/level.ts
import { BlockType } from "./block";
import { BlockSection } from "./blockSection";
import { EnemyType } from "./enemy";
import { Hero } from "./hero";
import { ItemType } from "./item";
import { ItemSectionType } from "./itemSection";
import { PipeType } from "./pipe";

export type Level = {
  level: string;
  pathToSpriteJson: string;
  width: number;
  height: number;
  timeLimit: number;
  song: string;
  hero: Hero;
  enemies: EnemyType[];
  items: ItemType[];
  itemSections: ItemSectionType[];
  blockSections: BlockSection[];
  blocks: BlockType[];
  pipes: PipeType[];
};
